wazo_confd/plugins/external_app/notifier.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from xivo_bus.resources.external_app.event import (
+    CreateExternalAppEvent,
+    DeleteExternalAppEvent,
+    EditExternalAppEvent,
+)
+
+from wazo_confd import bus
+
+from .schema import ExternalAppSchema
+
+ONLY_FIELDS = ['name']
```

Comment:
By default we return only the minimum information required in the message. (Mostly all confd event only return ID).
Since we don't use these events, I would not bother me with what we should send in the event. I want to we way to have a real use case / feature request from the community

Reply:
OK, thanks for the clarification.